packages/odyssey-react-mui/src/labs/AppTile.tsx

```diff
@@ -76,18 +80,28 @@ export type AppTileProps = {
     }
 );
 
+type LoadingTileProps = {
+  children: boolean;
+  description: boolean;
+  hasTopSection: boolean;
+  image: boolean;
+  overline: boolean;
+  title: boolean;
+};
+
 const ImageContainer = styled("div", {
-  shouldForwardProp: (prop) =>
-    prop !== "odysseyDesignTokens" && prop !== "hasAction",
+  shouldForwardProp: (prop) => prop !== "odysseyDesignTokens",
 })<{
   odysseyDesignTokens: DesignTokens;
-  hasAction: boolean;
-}>(({ odysseyDesignTokens, hasAction }) => ({
+}>(({ odysseyDesignTokens }) => ({
   display: "flex",
   alignItems: "flex-start",
   maxHeight: APP_TILE_IMAGE_HEIGHT,
   marginBlockEnd: odysseyDesignTokens.Spacing5,
-  paddingRight: hasAction ? odysseyDesignTokens.Spacing5 : 0,
+
+  ["& img"]: {
```

Comment:
Are we just trying to target the child `img` with this declaration? If so, I'm not sure this works as you expect. Should only need this right?
```css
img: {...
```